Cargo.toml

```
@@ -14,7 +14,11 @@ keywords = ["parity", "scale", "encoding", "decoding", "bits"]
 include = ["Cargo.toml", "src/**/*.rs", "README.md", "LICENSE"]
 
 [features]
-default = ["serde", "scale-info"]
+default = ["std", "serde", "scale-info"]
+std = [
+    "scale-info/std",
```

Can we only enable scale-info if it's explicitly requested, or was there a reason to not put the `?` on it here? 

```suggestion
    "scale-info?/std",
```

---

No, I missed that one. Thanks!